### PR TITLE
Initial attempt at HTMLv5 support

### PR DESCRIPTION
**Pull request by castaway**
TODO: Needs docs

Sending this as a PR to allow easy review/diffing - still needs docs/changelog/new version etc

### CODE REVIEW

**Comment by petdance** [comment]
Thanks for driving this.

> still needs docs/changelog/new version etc

And tests.

This will be version 5.0.0 when it goes out.

**Comment by castaway** [comment]
Any thoughts on what tests we can add? I've added a basic one to check whether the v5 tags get loaded if you ask them to.

**Comment by castaway** [comment]
Updated with version, changelog, docs explaining how to get v5 tags etc..

File: lib/HTML/Tagset/v4.pm [hunk 1]
```diff
@@ -1,4 +1,5 @@
-package HTML::Tagset::v4;
+package # Hide package from pause
```
Comment (petdance) [inline lib/HTML/Tagset/v4.pm]: s/pause/PAUSE/

**Comment by petdance** [comment]
This is looking really good.  Thanks again for making this happen.

I've created a new branch, `v5`.  Let's work against that, and then I can squash merge it (or you can re-PR it) down to dev.  There's code I'd like to add too but I want to keep the dev commit pretty clean.

**Comment by petdance** [comment]
Let's carry on discussion here: https://github.com/petdance/html-tagset/issues/2

**Pull request closed without merging**